Better issue templates (#237)
Direwolf20-MC/BuildingGadgets: .github/ISSUE_TEMPLATE/bug-report.md

---
name: Bug Report
about: Report a bug/crash

> title: '[Bug] A brief description about problem you found'

Reviewer: ```suggestion
title: ''
```
The only thing of value this adds is an encouragement to use [x] psudo-tags. Those are not necessary or wanted. We have real tags, and we can decide if what is being described is a bug (it's usually obvious).

labels: ''
assignees: ''

---

> <!--

Reviewer: ```suggestion

```

1. This makes all markdown in this text useless, since it shows no preview.
2. The likely reason someone wouldn't delete the template is because the used it instead. This just recently happened. In such a case, there would be no description, as it would have been discarded upon posting. I'd rather just edit their comment than them have to re-create it.

**IMPORTANT:** Before continuing, please ensure that Building Gadgets (and any other mods involved) is updated to the latest available version
----------------------------------------------------------------------------
After deleting this template, please provide the following information:
* Forge Version
* Building Gadgets version 

>   * **NEVER** put "lastest", new releases might come out as you post the issue

Reviewer: ```suggestion

```
This text should be kept to a minimum. The more the text, the less likely people are to read it (and to do so thoroughly). There has never been a problem with people typing 'latest', and it's not worth an extra line for the rare chance someone would in the future.

>   * If updating crashes the game, clearly state that in your description

Reviewer: ```suggestion

```
Since this is the only bullet point left, it can be incorporated into the "If the issue is a crash..." line. I've done that below.

>   * Use the version Forge shows you **In-Game**

Reviewer: ```suggestion

```
This doesn't add enough value to warrant being explicitly stated to everyone.

Reviewer: This is the last change that still needs implementing.

> * *If the issue is a crash*; An external link to the crash report

Reviewer: ```suggestion
* *If the issue is a crash*; An external link to the crash report, and state if it was caused by updating
```

* **How to reproduce the bug**
* An image about the bug

> -->

Reviewer: ```suggestion

```

----------------------------------------------------------------------------
After deleting this template, please provide the following information:
* Forge Version
* Building Gadgets version 
  * Use the version Forge shows you *In-Game*

> * *If the issue is a crash*; An external link to the crash report and the **action caused crash** (e.g. update mod)

Reviewer 2: shouldn't it be 
```suggestion
* *If the issue is a crash*; An external link to the crash report and the **action that caused the crash**
```
What do you mean with "e.g. update mod"? In theory updating the mod should not be the reason for the crash. And even if it only happens in a newer version, it won't help us much if people say "here's the crashreport and it happend after I updated the mod"

* **How to reproduce the bug**

> * An image about the bug

Reviewer 2: Could this be a little bit more specific? Reading this I'm unsure what you're referring to, which will reduce the likeliness for people to provide information.
